g2p/Cargo.toml

```diff
@@ -12,7 +12,7 @@ A crate to create types that implement fast finite field arithmetic.
 """
 categories = [ "no-std", "cryptography", "algorithms" ]
 keywords = [ "finite-field", "galois", "macro", "newtype"]
-rust-version = "1.61"
+rust-version = "1.70"
```

Comment:
See above.

Reply:
reverted back to 1.61 in [ca9be10](https://github.com/WanzenBug/g2p/pull/10/commits/ca9be102ae5ffe2966decb983ca69a6b87b71f1e)